test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -767,6 +771,10 @@ UCS_TEST_P(test_uct_cm_sockaddr, many_conns_on_client)
 
 UCS_TEST_P(test_uct_cm_sockaddr, err_handle)
 {
+    if (!strcmp(GetParam()->md_name.c_str(), "sockcm")) {
```

Comment:
minor: add common function called "skip_sockcm_cm()"

Reply:
i'll add. it's temporary anyway...

Reply:
there is another issue, CM should not be related to MD, it should be on top of component and component should be the same as sock transport. The same for rdmacm_cm, need to move it from rdmacm to ib component.

Reply:
yes but this is a large change that should be done (along with rdmacm) separately IMO.

Reply:
so, maybe complete stared things right way first?

Reply:
then our 2 cms will have different behavior. i think they should be changed together.

Reply:
I meant need to complete first then start to implement second in right way... but in this case we may have problems with deprecation.
On other hand, when we have 2 CM we need to match them to TL in UCP, by design it should be component index. Seems line we'll have to hardcode it somehow for migration period